lib/files.cf

```diff
@@ -483,7 +483,7 @@ bundle edit_line set_variable_values_ini(tab, sectionName)
 
       # match a line starting like the key something
       "$(index)\s*=.*"
-      edit_field => col("=","2","$($(tab)[$(sectionName)][$(index)])","set"),
+      edit_field => col("\s*=\s*","2","$($(tab)[$(sectionName)][$(index)])","set"),
```

Comment:
I can see this resulted in the RHS having a space after the = https://travis-ci.org/cfengine/masterfiles/jobs/622422779#L1247